scripts/artifacts-building/containers/build-process.sh

```diff
@@ -77,6 +77,12 @@ patch_all_roles
 
 # Run playbooks
 cd /opt/rpc-openstack/openstack-ansible/playbooks
+
+# The host must only have the base Ubuntu repository configured.
+# All updates (security and otherwise) must come from the RPC-O apt artifacting.
+openstack-ansible ${RPCD_DIR}/playbooks/configure-apt-sources.yml -e "host_ubuntu_repo=http://mirror.rackspace.com/ubuntu"
```

Comment:
The container artifacts will use mirror.rackspace.com as the default. See my comment below regarding the containers.